hedera-node/src/main/java/com/hedera/services/ServicesState.java

```diff
@@ -339,159 +227,176 @@ public void noMoreTransactions() {
-				accounts().getHash(),
-				storage().getHash(),
-				topics().getHash(),
-				tokens().getHash(),
-				tokenAssociations().getHash(),
-				diskFs().getHash(),
-				scheduleTxs().getHash(),
-				networkCtx().getHash(),
-				addressBook().getHash(),
-				runningHashLeaf().getHash(),
-				runningHashLeaf().getRunningHash().getHash(),
-				uniqueTokens().getHash()));
+		hashLogger.logHashesFor(this);
+		log.info(networkCtx());
 	}
 
 	public FCMap<MerkleEntityId, MerkleAccount> accounts() {
-		return getChild(ChildIndices.ACCOUNTS);
+		return getChild(StateChildIndices.ACCOUNTS);
 	}
 
 	public FCMap<MerkleBlobMeta, MerkleOptionalBlob> storage() {
-		return getChild(ChildIndices.STORAGE);
+		return getChild(StateChildIndices.STORAGE);
 	}
 
 	public FCMap<MerkleEntityId, MerkleTopic> topics() {
-		return getChild(ChildIndices.TOPICS);
+		return getChild(StateChildIndices.TOPICS);
 	}
 
 	public FCMap<MerkleEntityId, MerkleToken> tokens() {
-		return getChild(ChildIndices.TOKENS);
+		return getChild(StateChildIndices.TOKENS);
 	}
 
 	public FCMap<MerkleEntityAssociation, MerkleTokenRelStatus> tokenAssociations() {
-		return getChild(ChildIndices.TOKEN_ASSOCIATIONS);
+		return getChild(StateChildIndices.TOKEN_ASSOCIATIONS);
 	}
 
 	public FCMap<MerkleEntityId, MerkleSchedule> scheduleTxs() {
-		return getChild(ChildIndices.SCHEDULE_TXS);
+		return getChild(StateChildIndices.SCHEDULE_TXS);
 	}
 
 	public MerkleNetworkContext networkCtx() {
-		return getChild(ChildIndices.NETWORK_CTX);
+		return getChild(StateChildIndices.NETWORK_CTX);
 	}
 
 	public AddressBook addressBook() {
-		return getChild(ChildIndices.ADDRESS_BOOK);
+		return getChild(StateChildIndices.ADDRESS_BOOK);
 	}
 
 	public MerkleDiskFs diskFs() {
-		return getChild((ChildIndices.DISK_FS));
+		return getChild((StateChildIndices.DISK_FS));
 	}
 
 	public RecordsRunningHashLeaf runningHashLeaf() {
-		return getChild(ChildIndices.RECORD_STREAM_RUNNING_HASH);
+		return getChild(StateChildIndices.RECORD_STREAM_RUNNING_HASH);
 	}
 
 	public FCMap<MerkleUniqueTokenId, MerkleUniqueToken> uniqueTokens() {
-		return getChild(ChildIndices.UNIQUE_TOKENS);
+		return getChild(StateChildIndices.UNIQUE_TOKENS);
 	}
 
 	public FCOneToManyRelation<Integer, Long> uniqueTokenAssociations() {
-		return uniqueTokenAssociations;
+		return metadata.getUniqueTokenAssociations();
 	}
 
 	public FCOneToManyRelation<Integer, Long> uniqueOwnershipAssociations() {
-		return uniqueOwnershipAssociations;
+		return metadata.getUniqueOwnershipAssociations();
 	}
 
 	public FCOneToManyRelation<Integer, Long> uniqueTreasuryOwnershipAssociations() {
-		return uniqueTreasuryOwnedAssociations;
+		return metadata.getUniqueTreasuryOwnershipAssociations();
 	}
 
-	void setUniqueTokenAssociations(FCOneToManyRelation<Integer, Long> uniqueTokenAssociations) {
-		this.uniqueTokenAssociations = uniqueTokenAssociations;
+	private void internalInit(Platform platform, BootstrapProperties bootstrapProps) {
+		setImmutable(false);
```

Comment:
This is unnecessary, immutable is `false` by default`.

Reply:
Removed.